Main.java

```diff
@@ -0,0 +1,73 @@
+package com.company;
+
+import java.io.*;
+import java.util.StringTokenizer;
+
+public class Main {
+
+    public static void main(String[] args){
+
+        if(args.length!=2)
+        {
+            System.out.println("File path and search word cannot be empty");
+            System.exit(0);
+        }
+        String path=args[0];

```

Comment:
There should be a space before and after =
Here and everywhere else in the code

Reply:
Sure. Will do.